src/js/grapl-cdk/README.md

```diff
@@ -137,28 +150,34 @@ docker-machine create --driver "amazonec2" --amazonec2-private-address-only --am
 #
 
 # create a Docker Swarm cluster
-AWS01_IP=$(docker-machine ip aws01)
-eval $(docker-machine env aws01 --shell sh)
+AWS01_IP=$(docker-machine ip "$AWS01_NAME")
+eval $(docker-machine env "$AWS01_NAME" --shell sh)
 docker swarm init --advertise-addr $AWS01_IP
 
+
 # extract the join token
 WORKER_JOIN_TOKEN=$(docker swarm join-token worker -q)
 
 # make aws02 and aws03 join the swarm
-eval $(docker-machine env aws02 --shell sh)
+eval $(docker-machine env "$AWS02_NAME" --shell sh)
 docker swarm join --token $WORKER_JOIN_TOKEN "$AWS01_IP:2377"
-eval $(docker-machine env aws03 --shell sh)
+eval $(docker-machine env "$AWS03_NAME" --shell sh)
 docker swarm join --token $WORKER_JOIN_TOKEN "$AWS01_IP:2377"
 
-# start DGraph
+# get DGraph compose template
 cd $HOME
-wget https://github.com/dgraph-io/dgraph/raw/master/contrib/config/docker/docker-compose-multi.yml
-eval $(docker-machine env aws01 --shell sh)
+wget https://github.com/grapl-security/grapl/raw/staging/src/js/grapl-cdk/docker-compose-multi.yml
+sed -e 's/aws01/'"${AWS01_NAME}"'/g' -i docker-compose-multi.yml
```

Comment:
IMO rather than a `sed` here we should just parametrize the `docker-compose-multi.yml` with the environment variables. See e.g. what we do with tags here: https://github.com/grapl-security/grapl/blob/staging/docker-compose.yml#L59

Reply:
yeah that's way better

Reply:
addressed in later commit